### DISCUSSION TITLE
Your nice home page is gone ??

### DISCUSSION THREAD

**Discussion by lionpeloux** [squidfunk/mkdocs-material#4590] [General]
Your nice home page is gone : is this intentional ??
https://squidfunk.github.io/mkdocs-material/

<img width="1792" alt="Capture d’écran 2022-11-03 à 18 01 30" src="https://user-images.githubusercontent.com/951626/199786785-6fd51e3c-b4bd-471c-8c8e-1006cd2f62dd.png">

**Comment by squidfunk** [comment]
Thanks for reporting – very much unintentional, I'll fix it asap 😅

**Reply by squidfunk** [reply]
Fixed! I've simplified the whole theme development process, but deploying our own docs with Insiders, we have to jump through several hoops due to custom overrides. If you refresh your browser, it should now work again.
